src/collaboration/shared.js

```diff
@@ -210,7 +210,12 @@ module.exports = (name, id, crdtType, ipfs, collaboration, clocks, options) => {
     } else {
       const newState = crdtType.join.call(changeEmitter, state, s)
       if (crdtType.incrementalValue) {
-        valueCache = crdtType.incrementalValue(state, newState, s, valueCache)
+        if (!valueCache) {
+          const initial = crdtType.initial()
+          valueCache = crdtType.incrementalValue(initial, newState, newState, valueCache)
```

Comment:
I'm curious, what's the source of this issue?

If the problem is happening because the state is being loaded (in this line) and the value cache is empty, shouldn't we be calculating the incremental state right after the state being loaded?

Reply:
The cache could be initialized at load time... I made a new change where I modified the .start() method to initialize the cache when that is called.